test/Microsoft.AspNetCore.Mvc.Core.Test/PhysicalFileResultTest.cs

```diff
@@ -52,6 +52,144 @@ public void Constructor_SetsContentTypeAndParameters()
             MediaTypeAssert.Equal(expectedMediaType, result.ContentType);
         }
 
+        [Theory]
+        [InlineData(0, 3, "File", 4)]
+        [InlineData(8, 13, "Result", 6)]
+        [InlineData(null, 3, "File", 4)]
+        [InlineData(8, null, "ResultTestFile contents�", 26)]
```

Comment:
That's just how this file is. I am using the same one as other tests in this test class. Not sure about the purpose.